modules/openapi/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+plugins {
+    kotlin("jvm") version "2.0.0"
+    id("org.openapi.generator") version "6.2.1"
+}
+
+group = "com.sphereon.oid.fed"
+version = "1.0-SNAPSHOT"
+
+project.extra.set("openApiPackage", "com.sphereon.oid.fed.openapi")
+
+val profiles = project.properties["profiles"]?.toString()?.split(",") ?: emptyList()
+val isModelsOnlyProfile = profiles.contains("models-only")
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+}
+
+openApiGenerate {
+    val openApiPackage: String by project
+    generatorName.set("kotlin")
+    packageName.set("com.sphereon.oid.fed.openapi")
+    apiPackage.set("$openApiPackage.api")
+    modelPackage.set("$openApiPackage.models")
+    inputSpec.set("$rootDir/src/main/kotlin/com/sphereon/oid/fed/openapi/openapi.yaml")
```

Comment:
I had to update it to `    inputSpec.set("$rootDir/modules/openapi/src/main/kotlin/com/sphereon/oid/fed/openapi/openapi.yaml")
` for it to run

Reply:
Depends on where you are running the command prob

Reply:
I have the old problem "works on my machine" :smile:

Reply:
changing from rootDir to projectDir fixed the problem